docs/gathered-data.md

```diff
@@ -130,8 +130,12 @@ Id in config: oauths
 
 ## ClusterOperators
 
-collects all ClusterOperators and their resources.
-It finds unhealthy Pods for unhealthy operators
+collects all the ClusterOperators definitions and their resources.
+It finds degraded Pods for degraded cluster operators or that lives at the Cluster Operator's namespace, to collect:
```

Comment:
Some grammar question/suggestion:
It finds degraded Pods for degraded cluster operators or those that live at the Cluster Operator's namespace, to collect:

Reply:
We just agreed with Ricardo that using `unhealthy` is probably better. Because the CO can be just unavailable...and yeah pod is not degraded :)